lib/modules/manager/gomod/artifacts.ts

```diff
@@ -120,15 +120,16 @@ export async function updateArtifacts({
     logger.debug('No go.sum found');
     return null;
   }
-
   const goModDir = upath.dirname(goModFileName);
 
-  const vendorDir = upath.join(goModDir, 'vendor/');
-  const vendorModulesFileName = upath.join(vendorDir, 'modules.txt');
```

Comment:
keep it..easier to read

Reply:
Thanks for review.  The `vendorModulesFileName` is now kept.